timed/employment/tests/test_location.py

```diff
@@ -1,28 +1,56 @@
+import pytest
 from django.urls import reverse
 from rest_framework import status
 
-from timed.employment.factories import LocationFactory
-
-
-def test_location_list(auth_client):
-    LocationFactory.create()
+from timed.employment.factories import EmploymentFactory, LocationFactory
+from timed.projects.factories import CustomerAssigneeFactory
+
+
+@pytest.mark.parametrize(
+    "is_employed, is_customer_assignee, is_customer, expected",
+    [
+        (False, True, True, 0),
+        (False, True, False, 0),
+        (True, True, True, 1),
+        (True, True, False, 1),
+        (True, False, False, 1),
+    ],
+)
+def test_location_list(
+    auth_client, is_employed, is_customer_assignee, is_customer, expected
+):
+    location = LocationFactory.create()
+    if is_customer_assignee:
+        CustomerAssigneeFactory.create(user=auth_client.user, is_customer=is_customer)
+    if is_employed:
+        EmploymentFactory.create(user=auth_client.user, location=location)
```

Comment:
Use `setup_customer_and_employment_status()`

Reply:
Forgot that one, thanks